apiary-metastore-events/README.md

```diff
@@ -0,0 +1,17 @@
+# Apiary Metastore Events
+
+##  Overview
+The Apiary Metastore Events contains set of modules responsible for retrieving and processing of the Hive Metastore events.
```

Comment:
Small grammatical suggestions:
"The Apiary Metastore Events contains set of modules responsible for retrieving and processing of the Hive Metastore events."
to
"Apiary Metastore Events contains a set of modules responsible for retrieving and processing Hive Metastore events."